src/test/java/org/rulez/demokracia/pdengine/votecast/VoteCastAgainTest.java

```diff
@@ -0,0 +1,80 @@
+package org.rulez.demokracia.pdengine.votecast;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+import static org.rulez.demokracia.pdengine.testhelpers.CastVoteTestHelper.CHOICE_B;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.rulez.demokracia.pdengine.annotations.TestedBehaviour;
+import org.rulez.demokracia.pdengine.annotations.TestedFeature;
+import org.rulez.demokracia.pdengine.annotations.TestedOperation;
+import org.rulez.demokracia.pdengine.choice.RankedChoice;
+
+@TestedFeature("Vote")
+@TestedOperation("Cast vote")
+@TestedBehaviour(
+  "if there was a cast vote from the same user, the old one is deleted"
+)
+@RunWith(MockitoJUnitRunner.class)
+@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

Comment:
No need for this. Change the name of the method changeVoteOfUser to assertChangeVoteOfUser, and all is set.